gadget/update.go

```diff
@@ -425,6 +457,13 @@ func EnsureLayoutCompatibility(gadgetLayout *LaidOutVolume, diskLayout *OnDiskVo
 	return nil
 }
 
+type DiskEncryptionMethod string
+
+const (
+	EncryptionLUKS DiskEncryptionMethod = "LUKS"
+	EncryptionICE  DiskEncryptionMethod = "ICE"
```

Comment:
This could be a good place to remind that ICE stands for "Inline Crypto Engine".